Source/Core/DolphinQt/GameList/GameTracker.cpp

```diff
@@ -34,6 +34,8 @@ GameTracker::GameTracker(QObject* parent) : QFileSystemWatcher(parent)
 
   connect(this, &QFileSystemWatcher::directoryChanged, this, &GameTracker::UpdateDirectory);
   connect(this, &QFileSystemWatcher::fileChanged, this, &GameTracker::UpdateFile);
+  connect(&Settings::Instance(), &Settings::GameCoverUpdateRequested, this,
+          &GameTracker::RefreshAll);
```

Comment:
This does unnecessarily much work. You should just download banners, not reload everything.

Has this been tested to work, by the way? The way the cache works, I'm not sure if this actually will download any banners.

Reply:
It seemed to work in my tests.